formats/polus-ome-zarr-converter-plugin/plugin.json

```diff
@@ -1,45 +1,57 @@
+
```

Comment:
It's unclear to me why git is registering this as a complete rewrite of the manifest. Is it because a space was added at the beginning of the file?

Reply:
I was having issue with plugin manifest when registering plugin on WIPP. I had to rewrite it